### ISSUE TITLE
Reported bug when installing on Mac for the first time - unox tap failure

### ISSUE THREAD

**Issue by otakumike** [ironstar-io/tokaido#22]
**What happened**:
When running Tok the first time, it was uanble to install unison-fsmonitor using the automated brew tap:

```
unison-fsmonitor is missing. Tokaido will install it with Homebrew
Tokaido encountered a fatal error and had to stop at command 'brew install eugenmayer/dockersync/unox'
==> Installing unox from eugenmayer/dockersync
==> Downloading https://codeload.github.com/hnsl/unox/tar.gz/0.2.0
Error: Failure while executing; gunzip -q -N -- /private/tmp/d20180827-35333-1wet2uk/0.2.0 exited with 1.
2018/08/27 12:10:02 exit status 1
```

**What you expected to happen**:
Tokaido to automatically install dependencies

**How to reproduce it (as minimally and precisely as possible)**:
Run the Unox install steps manually (from [here](https://github.com/hnsl/unox))

**Anything else we need to know?**:
This can be resolved by installing the dependency manually. 

**Environment**:
- Tokaido version (use `tok version`): 1.0.0
- Drupal version: 8.5.6
- Operating system and version: MacOS

**Additional context**

**Comment by yjimk** [comment]
Looks like this relates to https://github.com/EugenMayer/homebrew-dockersync/issues/6

**Comment by yjimk** [comment]
Root cause may be recent update to brew: https://discourse.brew.sh/t/unpack-method-gzip-fails-for-a-specific-formula/2864

Will file GH issue on brew once the discourse post is addressed due to the issue not fitting the brew issue template

**Comment by otakumike** [comment]
Is it worth us still bringing the unison-fsmonitor binary directly into our package? It should fix this particular issue, and insulate us from further dependency-related issues?

**Comment by otakumike** [comment]
leaving this open until the root issue is resolved

**Comment by otakumike** [comment]
The root issue is resolved and 1.2.0 reverts to using brew to install unox
